CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ else ()
     set(LIB_EXTENSION ".dll")
     add_compile_definitions(_WINDOWS _WIN32 WIN32)
     add_compile_definitions(_WIN64)
+    add_compile_definitions(/DEBUG)
```

Comment:
I think `add_link_option('/DEBUG')` would do the trick as we want to modify the LINKFLAGS like the scons version